filereader/texttemplate/texttemplate.go

```diff
@@ -21,9 +22,27 @@ func Parse(filename string, funcs template.FuncMap) (*template.Template, error)
 			}
 			return content, nil
 		},
+		"toJSON": func(v interface{}) (string, error) {
```

Comment:
Thanks for the good catch 👍 Let me also add that both sprig's and helm's implementations of `toJson` seems to swallow errors, which sounds problematic to me. I'm now convinced to have our own implementation!